src/CMakeLists.txt

```diff
@@ -4,84 +4,84 @@ find_package(JNI REQUIRED)
 include(UseJava)
 
 set(SOURCES
-    jni_wrappers/call_object_method.cpp
-    jni_wrappers/call_static_method.cpp
-    jni_wrappers/environment.cpp
-    jni_wrappers/exception_check.cpp
-    jni_wrappers/find_class.cpp
-    jni_wrappers/method_id_cache.cpp
-    types/string.cpp
-    virtual_machine.cpp
-)
+        jni_wrappers/call_object_method.cpp
+        jni_wrappers/call_static_method.cpp
+        jni_wrappers/environment.cpp
+        jni_wrappers/exception_check.cpp
+        jni_wrappers/find_class.cpp
+        jni_wrappers/method_id_cache.cpp
+        types/string.cpp
+        virtual_machine.cpp
+        )
```

Comment:
I used previous way of formatting CMake files quite consistently, it would be good to keep it.

Reply:
Same comment to other CMake files.